Panels/tests/Header-specs.js

```diff
@@ -103,105 +88,53 @@ describe('Header Specs', () => {
 			</Header>
 		);
 
-		const actual = subject.find(`.${css.slotAfter}`).first().text();
+		const slotAfter = screen.getByText(expected).parentElement;
 
-		expect(actual).toBe(expected);
+		expect(slotAfter).toHaveClass(css.slotAfter);
 	});
 
 	test('should not render back button', () => {
-		const subject = mount(
-			<Header />
-		);
+		render(<Header />);
 
-		const backButton = subject.find(`.${css.slotBefore}`).find('Button');
-		const expected = 0;
-		const actual = backButton.length;
+		const backButton = screen.queryByLabelText('go to previous');
 
-		expect(actual).toBe(expected);
+		expect(backButton).toBeNull();
 	});
 
 	test('should render close button when \'noCloseButton\' is not specified', () => {
-		const subject = mount(
-			<Header />
-		);
+		render(<Header />);
 
-		const closeButton = subject.find(`.${css.slotAfter}`).find('Button');
-		const expected = 1;
-		const actual = closeButton.length;
+		const closeButton = screen.getByLabelText('Exit app');
 
-		expect(actual).toBe(expected);
+		expect(closeButton).toBeInTheDocument();
 	});
 
 	test('should not render close button when \'noCloseButton\' is set to true', () => {
-		const subject = mount(
-			<Header noCloseButton />
-		);
+		render(<Header noCloseButton />);
 
-		const closeButton = subject.find(`.${css.slotAfter}`).find('Button');
-		const expected = 0;
-		const actual = closeButton.length;
+		const closeButton = screen.queryByLabelText('Exit app');
 
-		expect(actual).toBe(expected);
+		expect(closeButton).toBeNull();
 	});
 
 	test('should call onClose when close button is clicked', () => {
 		const handleClose = jest.fn();
-		const subject = mount(
-			<Header onClose={handleClose} />
-		);
-
-		tap(subject.find(`.${css.slotAfter}`).find('Button'));
+		render(<Header onClose={handleClose} />);
 
-		const expected = 1;
-		const actual = handleClose.mock.calls.length;
+		fireEvent.mouseDown(screen.getByRole('button'));
+		fireEvent.mouseUp(screen.getByRole('button'));
```

Comment:
Is there a reason why you used these 2 lines and not `userEvent.click(screen.getByRole('button'))`?

Reply:
With userEvent.click, onTap will be called twice

![image](https://user-images.githubusercontent.com/63335068/126789882-779ffdaf-2575-4fd8-a739-04ec084b8397.png)